frontend/lib/src/components/widgets/Button/Button.tsx

```diff
@@ -63,4 +65,4 @@ function Button(props: Props): ReactElement {
   )
 }
 
-export default memo(Button)
+export default elementMemo<typeof Button, Props>(Button)
```

Comment:
One general consideration could be that we apply all the memo's directly in the `ElementNodeRenderer` instead of within the components. I think this could potentially reduce the need to add `elementHash` to the element props.

Reply:
Yeah that's a good thought! I am not sure whether this goes against any best-practice where the `memo` should happen in the export of the actual component, but your reasons make sense to me